Enigmatry.Entry.Blueprint.Infrastructure.Tests/Configuration/TestConfigurationBuilder.cs

```diff
@@ -45,29 +46,25 @@ public IConfiguration Build()
         };
 
         configurationBuilder.AddInMemoryCollection(dict);
+        _extraConfiguration?.Invoke(configurationBuilder);
         return configurationBuilder.Build();
     }
-
-    public IConfiguration BuildSchedulerConfiguration()
+    
+    public TestConfigurationBuilder WithSchedulerConfiguration()
```

Comment:
BuildSchedulerConfiguration had duplicated entry (we already had issue in tests in one of the projects because of that). I have changed to SchedulerConfiguration to be applied on top of initial configuration.